src/app/components/challenge/challengesubmit/challengesubmit.component.html

```diff
@@ -58,249 +58,250 @@
             <button class="btn ev-btn-dark btn-waves-effect waves-dark grad-btn
               code-btn" value="evalai push <image>:<tag> --phase <phase_name>"
               (click)="copyTextToClipboard(evalai_push)">
-              <i class="fa fa-clipboard" aria-hidden="true" alt="Copy to
-                    clipboard."></i>
+              <i class="fa fa-clipboard" aria-hidden="true" alt="Copy to clipboard.">
```

Comment:
Why do we need to change this file in `leaderboard page` PR? Also, only changes of re-ordering the code seem to be there. @Sanji515 Can you confirm?

Reply:
@lunayach There are not much changes here but formatted the code